src/d3a/models/market/one_sided.py

```diff
@@ -225,6 +239,9 @@ def accept_offer(self, offer_or_id: Union[str, Offer], buyer: str, *, energy: in
                 offer, buyer, original_offer, residual_offer
             )
 
+        # Delete the accepted_offer from self.offers:
```

Comment:
```suggestion
        # Delete the original offer from self.offers:
```

Reply:
Actually, I meant the accepted offer (i can remove the `_`). The accept_offer function trades the parsed offer . In the partial trade branch, offer is overwritten with accepted_offer. 
Let me know if this is clear.

Reply:
To me word `original` looks more clear than `accepted` since this offer is not fully accepted.